ci-operator/step-registry/openshift/e2e/test/qe/ui/destructive/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- jhou1
+- liangxia
+- jianzhangbjz
```

Comment:
@liangxia Shall I have UI team members as OWNERs here or keep current list?

Reply:
Since UI team owns this chain, so I'd prefer UI team take it over.

Reply:
okay, let me update